src/screens/Bridge/Exchange-CrossChain/CrossChain.tsx

```diff
@@ -179,6 +187,7 @@ function CrossChain({ fetchCrossChainTitle }: Props) {
   };
 
   const showLoading = buildTractionQuery.isLoading;
+  const ratesNotFound = toAsset && fromValue ? rate === 0 : false;
```

Comment:
Could this just be replaced by
```suggestion
  const ratesNotFound = toAsset && fromValue && rate === 0;
```